.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Self update testing
+on: push
+jobs:
+  # Checkout in separate job because docker image is alpine based and checkout action doesn't work.
+  functional:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: self-update-fixture
+    container:
+      image: quay.io/pantheon-public/php-ci:v7.4
+    name: Run functional tests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: self-update
+      - name: Checkout self-update-fixtures
+        uses: actions/checkout@v2
+        with:
+          repository: consolidation/self-update-fixture
+          path: self-update-fixture
+          ref: self-update
+      - name: Run Composer Install
+        run: composer install
+      - name: Add path repository
+        run: composer config repositories.self-update path ../self-update
+      - name: Require self-update package
+        run: COMPOSER_MIRROR_PATH_REPOS=1 composer require consolidation/self-update:"*"
+      - name: Phar Build
+        run: composer phar:build
+      - name: Get current version
+        run: ./wonder info | grep "1.2"
```

Comment:
Instead of putting the tests directly in ci.yml, make a bash script in the workflows directory to run that does these steps.

Reply:
@greg-1-anderson fixed. Please take another look